Week 1/Extra Practice Problems (Week 1)/0496: Next Greater Element I/dominic.cpp

```diff
@@ -0,0 +1,21 @@
+class Solution {
+public:
+    vector<int> nextGreaterElement(vector<int>& nums1, vector<int>& nums2) {
+        unordered_map<int, int> m;
```

Comment:
unordered_map is fine, but since all nums are small (<= 1e4), we can also just use a vector here